static/app/components/events/interfaces/breadcrumbs/utils.tsx

```diff
@@ -160,3 +161,48 @@ const USER_ACTIONS = [
 export function onlyUserActions(crumbs: Crumb[]): Crumb[] {
   return crumbs.filter(crumb => USER_ACTIONS.includes(crumb.type));
 }
+
+function moduleToCategory(module?: string | null) {
```

Comment:
I don't have enough context for how this method works to write a comment. None of my examples use it

I just moved it out of `static/app/components/events/interfaces/breadcrumbs/index.tsx` because `getVirtualCrumb()` depends on it.